### ISSUE TITLE
Default C# version for Mono framework

### ISSUE THREAD

**Issue by otebos** [dotnet/docs#21128]
[Enter feedback here]
Hi,
I'm wondering what are a default C# versions for Mono. Especially Mono 6.12.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 5694e62b-c756-1f62-4014-2ceafa54c011
* Version Independent ID: 3f01c378-22e4-197c-a862-ab4a924506c7
* Content: [C# language versioning - C# Guide](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/configure-language-version)
* Content Source: [docs/csharp/language-reference/configure-language-version.md](https://github.com/dotnet/docs/blob/master/docs/csharp/language-reference/configure-language-version.md)
* Product: **dotnet-csharp**
* Technology: **csharp-language-reference**
* GitHub Login: @BillWagner
* Microsoft Alias: **wiwagn**

**Comment by BillWagner** [comment]
I'd refer to the [Mono language version](https://www.mono-project.com/docs/about-mono/languages/csharp/) docs for that information. That indicates a partial implementation of C# 7.

Let me know if you need more assistance.

**Comment by otebos** [comment]
Yes, I saw that one but it last update was on on Apr 10, 2018 and by this time Mono was in version 5.8/5.10 and the C# version at this time was 7.3. 

The Wikipedia page on Mono ends history on version 6.0 with annotation "C# compiler defaults to version C# 8.0 RC" ([https://en.wikipedia.org/wiki/Mono_(software)]).

Now we have 6.12 so I'd assume it supports newest C# features ....

**Comment by BillWagner** [comment]
Ping @marek-safar for the best person to answer.

**Comment by marek-safar** [comment]
https://www.mono-project.com/docs/about-mono/languages/csharp/ is the most accurate status C# support in mcs (Mono C# compiler). However, recent Mono versions ship with csc (Roslyn C# compiler) as the default C# compiler. You can see which version of Roslyn is available in each Mono release at https://www.mono-project.com/docs/about-mono/versioning/#mono-source-versioning
